web/next.config.js

```diff
@@ -44,7 +48,7 @@ const nextConfig = {
 
   reactStrictMode: true,
   images: {
-    domains: ["world-id-public.s3.amazonaws.com"],
```

Comment:
let's add a note that `world-id-public.s3.amazonaws.com` is deprecated and should be removed soon